gundi_client_v2/client.py

```diff
@@ -36,31 +36,43 @@ async def post_observations(self, data: List[dict]) -> dict:
     async def post_events(self, data: List[dict]) -> dict:
         return await self._post_data(data=data, endpoint="events")
 
-    async def _post_data(self, data: List[dict], endpoint: str) -> dict:
+    async def post_event_attachment(self, event_id: str, attachment: dict) -> dict:
```

Comment:
I imagine the reason for the `attachment` argument to be a dict is to allow the user to pass more than one attachment. Then maybe we can use the plural "attachments", both for the argument and the method name?

Reply:
Actually, it is a dict because, according to [this httpx doc](https://www.python-httpx.org/quickstart/#sending-multipart-file-uploads), we need to send the binary image in a dict like this:

`{'upload-file': <IMAGE_IN_BYTES>}`

I tested and we can't send several files at the same time. I imagine we can manage this from outside the `gundi-client `and send several calls to `post_event_attachment` if needed

Reply:
I added a new version for supporting multiple files in a single request